test/test_build.py

```diff
@@ -81,17 +81,23 @@ def image_type_fixture(shared_tmpdir, build_container, request, force_aws_upload
     container_ref, images, target_arch, local = parse_request_params(request)
 
     if local:
-        cont_tag = "localhost/cont-base-" + "".join(random.choices(string.digits, k=12))
+        container_ref = "localhost/cont-base-" + "".join(random.choices(string.digits, k=12))
 
         # we are not cross-building local images (for now)
-        request.param = ",".join([cont_tag, images, "", "true"])
+        request.param = ",".join([container_ref, images, "", "true"])
 
-        # copy the container into containers-storage
-        subprocess.check_call([
-            "skopeo", "copy",
-            f"docker://{container_ref}",
-            f"containers-storage:[overlay@/var/lib/containers/storage+/run/containers/storage]{cont_tag}"
-        ])
+    # copy the container into containers-storage
+    # for all builds
+    arch = "amd64"
+    if target_arch == "aarch64" or target_arch == "arm64":
+        arch = "arm64"
```

Comment:
silly question, dosn't podman understand both kernel and golang arch names?

Reply:
Yes it seems to:
```
$ podman run --arch=aarch64 alpine arch
aarch64
$ podman run --arch=arm64 alpine arch
aarch64
```

Reply:
Perhaps we should just pass down target_arch to Podman and allow it to handle it or throw an error.

Reply:
Cool thanks, I didn't even think to just let podman handle it. Makes a lot of sense

Reply:
Turns out skopeo doesn't understand both. I will change to use podman pull instead